packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -342,11 +352,41 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     spotlightedParticipants
   ]);
 
+  /* @conditional-compile-remove(media-access) */
+  const meetingMediaAccess = useSelector(getMediaAccessSetting);
+  /* @conditional-compile-remove(media-access) */
+  const onToggleParticipantMicPeoplePaneProps = useMemo(() => {
+    return {
+      onForbidAudio: capabilities?.forbidOthersAudio?.isPresent ? onForbidAudio : undefined,
```

Comment:
Can we write a composite selector to calculate this data? if we use the capabilities in the selector like we are here this function this can be a lot simpler reading from that selector return

Reply:
Selector is more about filter call data like capatilities, participants and other call states.
This is about show or hide UI items base on capabilities, onToggleParticipantMicPeoplePaneProps is following the current pattern similar to spotLight, muteParticipant and others. 

![image](https://github.com/user-attachments/assets/ef938a12-7b25-4979-a94f-fc3d0a32aad1)

Reply:
Discussed, will create two consts instead of using capabilities?.forbidOthersAudio?.isPresent

Reply:
Done